Source/SwiftLintFramework/Rules/Style/AttributesRule.swift

```diff
@@ -293,28 +293,29 @@ public struct AttributesRule: ASTRule, OptInRule, ConfigurationProviderRule {
     private func parseAttributes(dictionary: SourceKittenDictionary) -> [SwiftDeclarationAttributeKind] {
         let attributes = dictionary.enclosedSwiftAttributes
         let blacklist: Set<SwiftDeclarationAttributeKind> = [
-            .mutating,
-            .nonmutating,
-            .lazy,
             .dynamic,
+            .fileprivate,
             .final,
             .infix,
+            .internal,
+            .lazy,
+            .mutating,
+            .nonmutating,
+            .open,
             .optional,
             .override,
             .postfix,
             .prefix,
-            .required,
-            .weak,
             .private,
-            .fileprivate,
-            .internal,
             .public,
-            .open,
-            .setterPrivate,
+            .required,
+            .rethrows,
```

Comment:
The addition of this kind is the only change in this file, but I resorted the collection literal to make it easier to update in the future.